client/src/pages/Household.js

```diff
@@ -10,29 +12,129 @@ import ButtonContainer from '../components/ButtonContainer';
 import Form from '../components/Form';
 import Wrapper from '../components/Wrapper';
 import PageContainer from '../components/PageContainer';
+import styled from '@emotion/styled';
+
+const DeleteButton = styled.img`
+  display: flex;
+  justify-content: center;
+  width: 20px;
+  cursor: pointer;
+  border: none;
+  &:focus {
+    outline: none;
+  }
+`;
 
 function Household() {
+  const [name, setName] = useState('');
+  const [rooms, setRooms] = useState([]);
+  const [members, setMembers] = useState([]);
+  const [member, setMember] = useState('');
+  const [room, setRoom] = useState('');
+
+  function addToRoomList(newRoom) {
+    const newItem = [...rooms, newRoom];
+    setRooms(newItem);
+  }
+
+  function addToMemberList(newMember) {
+    const newItem = [...members, newMember];
+    setMembers(newItem);
+  }
+
+  const handleSubmitRoom = (event) => {
+    event.preventDefault();
+    if (!room) return;
+    addToRoomList(room);
+    setRoom('');
+  };
+
+  const handleSubmitMember = (event) => {
+    event.preventDefault();
+    if (!member) return;
+    addToMemberList(member);
+    setMember('');
+  };
+
+  const handleDeleteRoom = (index) => {
+    const newRooms = [...rooms];
+    newRooms.splice(index, 1);
+    setRooms(newRooms);
+  };
+
+  const handleDeleteMember = (index) => {
+    const newMembers = [...members];
+    newMembers.splice(index, 1);
+    setMembers(newMembers);
+  };
+
   return (
     <PageContainer>
       <Wrapper>
         <Header />
         <PageCaption>Your household</PageCaption>
       </Wrapper>
-      <Form>
-        <InputContainer>
-          <InputCircle type="text" placeholder="Enter householdname" />
-        </InputContainer>
-        <InputContainer>
-          <InputCaption>Create household</InputCaption>
-          <InputLine type="text" placeholder="Add room" />
-        </InputContainer>
-        <InputContainer>
-          <InputCaption>Household members</InputCaption>
-          <InputLine type="text" placeholder="Add name" />
-        </InputContainer>
-      </Form>
+      <InputContainer>
+        <InputCircle
+          value={name}
+          onChange={(event) => setName(event.target.value)}
+          placeholder="Enter householdname"
+        />
+      </InputContainer>
+      <InputContainer>
+        <InputCaption>Create household</InputCaption>
+        <Form onSubmit={handleSubmitRoom}>
+          <InputLine
+            value={room}
+            onChange={(event) => setRoom(event.target.value)}
+            placeholder="Add room"
+          />
+        </Form>
+        {rooms.map((room, index) => (
+          <>
+            <span key={index} room={room}>
+              {room}
+            </span>
+            <DeleteButton
+              src={DeleteIcon}
+              onClick={() => {
+                handleDeleteRoom(index);
```

Comment:
It's fine to use the index to remove an element from an array.
An alternative is to use the room as the parameter and modify your `handleDelete` function.

```suggestion
                handleDeleteRoom(room);
```